docs/api/mount/hasAttribute.md

```diff
@@ -1,17 +1,14 @@
-# hasAttribute(attribute, value)
```

Comment:
Uhhh...I am a bit confused. You said remove hasAttribute from docs. Now we have deprecated the usage of `hasAttribute(attribute, value)`, and we should keep the detail on it?